test task counting minutes.js

```diff
@@ -18,5 +18,6 @@ let day = hour / 24;
      alert(('0' + min % 60).slice(-2) + "min");
   }
 }
-GetNum()
+GetNum();
```

Comment:
Let's change GetNum() to FormatDate(min) and push minutes to function directly without prompting user.
So you can call it like FormatDate(125)